.github/workflows/main.yml

```diff
@@ -47,3 +48,10 @@ jobs:
         env:
           CYPRESS_baseUrl: http://localhost:8080/
           NODE_ENV: offline
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This might not kick in until merged as secrets are not exposed to forks.

Annoyingly GitHub doesn't raise an error if a template variable is unset, it just gives you an empty string.

Reply:
:+1: I thought it was working for the other coverage report (I copied this block from the existing line for unit tests coverage)